desktop/cmp/panel/Panel.ts

```diff
@@ -173,15 +177,15 @@ export const [Panel, panel] = hoistCmp.withFactory<PanelProps>({
             };
 
             coreContents = vframe({
-                style: {display: collapsed ? 'none' : 'flex'},
+                style: {display: isRenderedCollapsed ? 'none' : 'flex'},
                 items: Children.toArray([
                     parseToolbar(tbar),
                     ...castArray(children),
                     parseToolbar(bbar)
                 ])
             });
         }
-        if (!collapsed) wasDisplayed.current = true;
+        if (!isRenderedCollapsed) wasDisplayed.current = true;
```

Comment:
I want to look a bit more closely at this + interaction with refreshContext - I think that this change is an improvement there, in that modal panels will get refreshed - but didn't fully trace out that part of panelModel